src/item.py

```diff
@@ -0,0 +1,13 @@
+class Item():
+  def __init__(self, name , description):
+    self.name = name
+    self.descript = description
+
+  def on_take(self, item):
+    print(f"You have picked up {item}")
+
+  def on_drop(self, item):
```

Comment:
The on drop method should be in the Player class in order to remove that item from the players items list. There should also be an add item method in the Room class to add that item to a room the player is currently in.